### ISSUE TITLE
__webpack_modules__[moduleId] is not a function

### ISSUE THREAD

**Issue by ezawadzki** [wwan5803/react-modal-resizable-draggable#43]
Hi,

When I try to install the last version, I got : 

`warning " > react-motion@0.5.2" has incorrect peer dependency "react@^0.14.9 || ^15.3.0 || ^16.0.0".`

Then I tried to start my app and just import the module, and got this webpack error in browser :

```
TypeError: __webpack_modules__[moduleId] is not a function
__webpack_require__
node_modules/react-modal-resizable-draggable/build/main.js:557
```

Thanks for you help ! I would like to use your module which seems great !

Emilie

**Comment by wwan5803** [comment]
maybe try the previous version 0.1.4?

**Comment by ryoman81** [comment]
The same problem of v0.1.5. react@16.14.0, react scripts@4.0.3.
Restored to v0.1.4, the problem resolved.

**Comment by Michael101sh** [comment]
This also happens to me in version 0.1.4.

**Comment by emrahyt** [comment]
l am facing with same problem. Is there any solution up to now?

**Comment by cfausn** [comment]
I also ran into this problem on v0.1.5  react @17.0.2, react-scripts @4.0.3
Restored to v0.1.4 and seems to be working fine.

**Comment by wwan5803** [comment]
The issue has been fixed in v0.1.6
